scripts/perftune.py

```diff
@@ -614,14 +618,18 @@ def __dev_is_hw_iface(self, iface):
     def __check_dev_is_bond_iface(self):
         if not os.path.exists('/sys/class/net/bonding_masters'):
             return False
-
-        return any([re.search(self.nic, line) for line in open('/sys/class/net/bonding_masters', 'r').readlines()])
+        bonddict={}
+        for nic in self.nics:
+            bonddict[nic]=any([re.search(nic, line) for line in open('/sys/class/net/bonding_masters', 'r').readlines()])
+        return bonddict
 
     def __learn_slaves(self):
-        if self.nic_is_bond_iface:
-            return list(itertools.chain.from_iterable([ line.split() for line in open("/sys/class/net/{}/bonding/slaves".format(self.nic), 'r').readlines() ]))
+        slaveslistpernic={}
+        for nic in self.nics:
+            if self.nic_is_bond_iface(nic):
+                slaveslistpernic[nic]=list(itertools.chain.from_iterable([ line.split() for line in open("/sys/class/net/{}/bonding/slaves".format(nic), 'r').readlines() ]))
```

Comment:
Here it's the same style disaster again: missing spaces around "=" and extra spaces after "[" and before "]".